### PR TITLE
Added a section on Changing the Web Console URL Port and Certificates

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1593763

@spadgett @rjhowe PTAL

### CODE REVIEW

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -896,3 +895,28 @@ xref:../install/index.adoc#install-planning[inventory file]:
...
+----
+
+To ensure users are correctly redirected, update the `PublicUrls` for the
+`openshift-web-console configmap`:
+
+----
+$ oc edit configmap/webconsole-config -n openshift-web-console
+----
+
+Update the values for `consolePublicURL` and `masterPublicURL`.
```
Comment (spadgett) [inline install_config/web_console_customization.adoc]: It's unclear to me if you're changing both master public URL or just console URL. There are probably more changes needed if changing master public URL. If you haven't changed master URL, the `masterPublicURL` property in the config map doesn't need to be updated.

Do we have any existing topics that cover changing master public URL?

**Review by spadgett — COMMENTED** [review]
cc @deads2k

File: install_config/web_console_customization.adoc (hunk 1 above)
Comment (ahardin-rh) [reply install_config/web_console_customization.adoc]: @spadgett I don't see much mentioned about changing the master public URL. It's only mentioned briefly in places like https://docs.openshift.com/container-platform/3.9/install_config/master_node_configuration.html#master-config-oath-authentication-config and https://docs.openshift.com/container-platform/3.9/install_config/master_node_configuration.html#master-config-asset-config

Comment (ahardin-rh) [reply install_config/web_console_customization.adoc]: @spadgett @deads2k  Any more thoughts?

Comment (ahardin-rh) [reply install_config/web_console_customization.adoc]: @spadgett @deads2k Bump

Comment (ahardin-rh) [reply install_config/web_console_customization.adoc]: @deads2k Okay, thanks! This is updated.

**Comment by ahardin-rh** [comment]
@yanpzhan Please provide QE review.
@openshift/team-documentation Please peer review.

**Review by kalexand-rh — COMMENTED** [review]
Couple of questions and suggestions

File: install_config/web_console_customization.adoc [hunk 2]
```diff
@@ -11,7 +11,6 @@
 
 toc::[]
 
-ifdef::openshift-origin,openshift-enterprise[]
```
[the remark this answers is not in the document]
Comment (ahardin-rh) [reply install_config/web_console_customization.adoc]: Yeah, not sure why this ifdef (with no endif) was in the doc. This cleans it out. The _topic_map.yml already defines that this topic should be in Origin and OCP docs only, so this like is unnecessary.

**Comment by rjhowe** [comment]
With the new web console being in a pod, in the master-config.yaml the values for masterPublicURL oauthConfig.assetPublicURL and oauthConfig.masterPublicURL need to a the same and the namedcertificate needs to configured for these too, as we get redirect to these URLs for paths /oauth/authorize and /console/oauth.

File: install_config/web_console_customization.adoc [hunk 3]
```diff
@@ -922,3 +921,28 @@ xref:../install/index.adoc#install-planning[inventory file]:
 # See: https://docs.openshift.com/enterprise/latest/install_config/aggregate_logging.html
 #openshift_master_logging_public_url=https://kibana.example.com
 ----
+
+[[changing-the-web-console-url-port-and-certificates]]
+== Changing the Web Console URL Port and Certificates
+
+To ensure your custom certificate is served when users access the web console
+URL, add the certificate and URL to the `NamedCertificates` section of the
```
Comment (DanyC97) [inline install_config/web_console_customization.adoc]: typo `NamedCertificates` vs `namedCertificates` ?

 as per the [code](https://github.com/openshift/openshift-ansible/blob/006d82982d1dd3117bf9057ae6172a9d16380e0c/roles/openshift_control_plane/templates/master.yaml.v1.j2#L193)

**Review by DanyC97 — CHANGES_REQUESTED** [review]

**Pull request merged**